ui/src/components/RepoHeader.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, {Component} from 'react';
+import {FormattedHTMLMessage, defineMessages, injectIntl} from 'react-intl';
+import {Link} from "react-router-dom";
+
+const messages = defineMessages({
+  productName: {
+    id: 'RepoHeader.productName',
+    defaultMessage: 'Google Person Finder',
```

Comment:
Can be considered later but this name should be configurable. Should be in a different name if a third party runs Person Finder.

Reply:
That's a good point. I'll slap a TODO on this for now and do some thinking about how to handle white-labeling more generally, including in the React app.